imports/api/UserMonitor/detectors/methods.js

```diff
@@ -75,7 +81,8 @@ const keyvalues2vardecl = function (obj) {
   let vardecl = [];
```

Comment:
- Add a description of the function above
- Comments on what lines 84 and 85 are doing